packages/hash/api/src/graph/system-org.ts

```diff
@@ -0,0 +1,123 @@
+import {
+  SYSTEM_ACCOUNT_NAME,
+  SYSTEM_ACCOUNT_SHORTNAME,
+} from "@hashintel/hash-shared/environment";
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { types } from "@hashintel/hash-shared/types";
+import { getEntities } from "@hashintel/hash-subgraph/src/stdlib/element/entity";
+import {
+  extractEntityUuidFromEntityId,
+  Subgraph,
+  SubgraphRootTypes,
+} from "@hashintel/hash-subgraph";
+import { extractBaseUri } from "@blockprotocol/type-system";
+import { OrgModel, OrgSize } from "../model";
+import { PageDefinition, seedPages } from "../seed-data/seed-pages";
+
+// eslint-disable-next-line import/no-mutable-exports
+export let systemOrgAccountId: string;
```

Comment:
I know we already have some in the form of the `types` variable, but these global mutable variables do concern me :/ I think this is definitely something we should look into soon, perhaps next week (almost certainly out of scope for this PR), but generally these are an anti-pattern.

Reply:
Possibly not worth doing next week though. @CiaranMn is starting to look into TS type generation from schemas, which I've thought for a while could be used for the system types through a codegen step. It wouldn't help solve the `AccountId` thing, but ideally could solve the other bits to do with what should basically be static schemas which we're currently programmatically generating on every import of that file of the shared package 🙃

Reply:
I started out the implementation by having the `systemOrgAccountId` be returned by the `ensureSystemOrgAccountIdExists` method, and pass it explicitly as a parameter. Although I would prefer this approach, it exploded the scope of this PR because `systemAccountId` is being used in some quite hard to reach places, and the function arguments would have to be updated in a lot of places.

I am in favour of us more explicitly passing things around, and not relying on these uninitialised exports. It'd help ensure that things happen in the correct order.